example/ios/PagerViewExample.xcodeproj/project.pbxproj

```diff
@@ -600,10 +600,7 @@
 				);
 				OTHER_LDFLAGS = (
 					"$(inherited)",
-					"-Wl",
```

Comment:
why those flags have been removed?

Reply:
I checked, and installing pods is what removes them on my machine - not only on this branch but on the latest next as well. Do you want me to investigate it further? (And if not, should I bring them back or keep them removed?) 
Thanks!

Reply:
On my machine `-Wl` flag is present. Let's maybe not remove it now and we'll see if that issue appears in future?

Reply:
@krozniata sure, I brought all of them back now just in case, let's observe what happens going forward and we can always remove them later on.